hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java

```diff
@@ -447,4 +497,31 @@ private OMClientResponse createKeyErrorResponse(@Nonnull OMMetrics omMetrics,
     }
   }
 
+  /**
+   * Check Acls for the ozone object.
+   * @param ozoneManager
+   * @param volume
+   * @param bucket
+   * @param key
+   * @param checkKeyAccess
+   * @throws IOException
+   */
+  protected void checkKeyAcls(OzoneManager ozoneManager, String volume,
```

Comment:
Suggest to refactor into two separate APIs one for checkKeyAccess and one for checkBucketAccess to avoid confusion.